.setup/INSTALL_SUBMITTY_HELPER_BIN.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+########################################################################################################################
+########################################################################################################################
+# COPY VARIOUS SCRIPTS USED BY INSTRUCTORS AND SYS ADMINS FOR COURSE ADMINISTRATION
+
+echo -e "Copy the non-root scripts"
+
+if [ -z ${SUBMITTY_INSTALL_DIR+x} ]; then
```

Comment:
why did this get closed as outdated?

Reply:
the `-z` tests for not empty (null or otherwise). The variable is then read as `${VARIABLE+[word]}` which is that if `${VARIABLE}` is null/empty, replace it with nothing, else replace it with `[word]` (or in this case, `x`).